docs/resources/organizations_account.md

```diff
@@ -0,0 +1,48 @@
+---
+subcategory: "Organizations"
+---
+
+# huaweicloud_organizations_account
+
+Manages an Organizations account resource within HuaweiCloud.
```

Comment:
it's better to add an note message to describe this resource will not delete the account